packages/api/bin/serveUtils.js

```diff
@@ -82,12 +193,57 @@ async function addReconciliationReports(reconciliationReports) {
   );
 }
 
+async function addGranulesExecutions(granulesExecutions) {
+  const knex = await getKnexClient({
+    env: {
+      ...envParams,
+      ...localStackConnectionEnv,
+    },
+  });
+
+  const collectionPgModel = new CollectionPgModel();
+  return await Promise.all(
+    granulesExecutions.map(async (ge) => {
+      // Fetch the Collection ID
+      const [name, version] = ge.granule.collectionId.split('___');
```

Comment:
I would use the `deconstructCollectionId` helper from the `message` package

Reply:
Updated to use `deconstructCollectionId`.